django/contrib/auth/tests/test_auth_backends.py

```diff
@@ -34,8 +35,8 @@ class BaseModelBackendTest(object):
     backend = 'django.contrib.auth.backends.ModelBackend'
 
     def setUp(self):
-        self.curr_auth = settings.AUTHENTICATION_BACKENDS
-        settings.AUTHENTICATION_BACKENDS = (self.backend,)
+        self.curr_auth = copy(settings.AUTHENTICATION_BACKENDS)
```

Comment:
@timgraham  That's because we've changed the datatype to lists. Doing `self.curr_auth = settings.AUTHENTICATION_BACKENDS` an initial copy of the settings for the duration of the test was stored, and then it was replaced of the backup. As you can imagine, this worked for tuples since += created a new object, wheras doing += on lists messed with the backup copy too.